docs/bpmn-support.adoc

```diff
@@ -287,11 +287,11 @@ The default rendering uses `white` as fill color and `black` as stroke color.
 
 
 
-=== Text Annotation
+=== Artifacts
```

Comment:
According the BPMN specification, only **Text annotation** & **Group** are **Artifact**, and not **Text annotation association**.

<img width="860" alt="image" src="https://user-images.githubusercontent.com/4921914/87532589-ae995d00-c693-11ea-9bca-5c40f6a747ce.png">

Reply:
I would rather have created a new **Association** section:

<img width="860" alt="image" src="https://user-images.githubusercontent.com/4921914/87532850-09cb4f80-c694-11ea-9fef-c5ad044dd7ab.png">

Reply:
Or put in **Flow** section.

Reply:
OK I agree,
I have added an Association section